pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/InvertedIndexCreator.java

```diff
@@ -36,38 +36,28 @@
  * |                           Data for last bitmap                          |
  * |-------------------------------------------------------------------------|
  * </pre>
+ *
+ * <p>To create an inverted index:
+ * <ul>
+ *   <li>1. Construct an instance of <code>InvertedIndexCreator</code></li>
+ *   <li>2. Call add() for each docId to add dictId (dictIds for multi-valued column) into the creator</li>
```

Comment:
nit: call add() in sequece starting with docid 0

Reply:
Fixed.